samples/XCT.Sample/Pages/Views/Popups/TransparentPopup.xaml

```diff
@@ -1,19 +1,21 @@
 ﻿<?xml version="1.0" encoding="utf-8" ?>
-<xct:Popup xmlns="http://xamarin.com/schemas/2014/forms"
-           xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
-           xmlns:xct="http://xamarin.com/schemas/2020/toolkit"
-           xmlns:local="clr-namespace:Xamarin.CommunityToolkit.Sample.Pages.Views.Popups"
-           xmlns:iosxct="clr-namespace:Xamarin.CommunityToolkit.UI.Views.iOSSpecific;assembly=Xamarin.CommunityToolkit"
-           xmlns:uwpxct="clr-namespace:Xamarin.CommunityToolkit.UI.Views.WindowsSpecific;assembly=Xamarin.CommunityToolkit"
-           Size="{x:Static local:PopupSize.Tiny}"
-           Color="Transparent"
-           iosxct:Popup.ArrowDirection="Up"
-           uwpxct:Popup.BorderColor="Transparent"
-           x:Class="Xamarin.CommunityToolkit.Sample.Pages.Views.Popups.TransparentPopup">
+<xct:Popup
+    x:Class="Xamarin.CommunityToolkit.Sample.Pages.Views.Popups.TransparentPopup"
+    xmlns="http://xamarin.com/schemas/2014/forms"
+    xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+    xmlns:local="clr-namespace:Xamarin.CommunityToolkit.Sample.Pages.Views.Popups"
+    xmlns:xct="http://xamarin.com/schemas/2020/toolkit"
+    Size="{x:Static local:PopupSize.Tiny}"
```

Comment:
Totally unrelated, I think we can create an extension for this, right? I remember us doing it for something else, that will make this syntax better

Reply:
What kind of markup extension are you thinking? The `PopupSize` is a helper for the sample project to make it easier to specify size.

Are you thinking about making this something that consumers of XCT could use for their Popups? That may be a really useful API.